lib/matplotlib/animation.py

```diff
@@ -1199,9 +1199,16 @@ def _setup_blit(self):
         # axes
         self._blit_cache = dict()
         self._drawn_artists = []
+        # _post_draw needs to be called first to initialize the renderer
+        self._post_draw(None, self._blit)
+        # Then we need to clear the Frame for the initial draw
+        # This is typically handled in _on_resize because QT and Tk
+        # emit a resize event on launch, but the macosx backend does not,
+        # thus we force it here for everyone for consistency
+        self._init_draw()
```

Comment:
Would it make sense to make the osx backend instead conform to the other backends and emit the expected resize event?

Reply:
I considered this, but decided against it here. That is definitely a possibility, but I would consider those resize_events to actually be the incorrect piece. Each backend produced a differing number of the events, and seemingly in the future could be patched to remove some of the unnecessary resizes.

Reply:
Fair enough.